lib/task-data/tasks/secure-erase-drive.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     injectableName: 'Task.Drive.SecureErase',
     implementsTask: 'Task.Base.SecureErase',
     options: {
-        eraseSettings: []
+        eraseSettings: [],
+        baseUri: '{{ file.server }}'
```

Comment:
I would also suggest to replace ```baseUri``` with ```notificationUri``` and append the sub-router into the tail, so that the API uri will be limited in RackHD knowledge scope. If the notification API changed in the API in the future, people will not need to modify the secure erase. python script. Not a  blocking comment anyway.